client/blocks/get-apps/apps-badge.scss

```diff
@@ -2,9 +2,17 @@
 	display: flex;
 	max-width: 135px;
 	max-height: 40px;
-	overflow: hidden;
 	margin-left: 0.5rem;
 
+	// Set overflow: hidden here instead of the parent node so that its focus outline is not cut-off.
+	> a {
+		overflow: hidden;
+
+		&:focus {
```

Comment:
I think using `:focus-visible` is better for this a11y issue if the browser supports

Reply:
Added the suggested focus states :-)